news/2024/2024-06-04-mappers-guild-priority-quests-summer-2024.md

```diff
@@ -0,0 +1,106 @@
+  - Tier 4 ![platinum icon](/wiki/shared/news/2023-08-29-changes-to-the-mappers-guild/platinum-small.png) (hardest, least common, and highest reward)
+- **Multiple users can complete the same quest**. Most of the time, anyone who accomplishes the quest's objective before the deadline is eligible.
+
+By completing priority quests, you can earn some special rewards too:
+
+- All the usual [Mappers' Guild rewards](/wiki/Community/Mappers_Guild#rewards) for ranking beatmaps
+- A [**customised keychain**](/wiki/shared/news/2023-08-29-changes-to-the-mappers-guild/keychain.jpg) based on a pattern in your map
+- A profile badge upon completing 3 priority quests
+
+![Priority quest badge](/wiki/shared/news/2023-08-29-changes-to-the-mappers-guild/questbadge.png)
+
+## New priority quests
+
+**6** new quests have just been added to the [priority quest listing](https://mappersguild.com/missions), all of which will be open to mappers until **September 14, 2024**!
+
+Below is an overview of each quest. If any are confusing, feel free to ask questions in the comments!
+
+### ["new" beginnings](LINK) ![bronze icon](/wiki/shared/news/2023-08-29-changes-to-the-mappers-guild/bronze-small.png)
+
+Tons of mappers have Ranked maps, but a few have *never* touched Featured Artist songs.
+
+It's time to undergo the Mappers' Guild initiation: rank your **first Featured Artist map** using any song from [osu!'s music library](https://osu.ppy.sh/beatmaps/artists)!
+
+If you already have a Ranked map using a Featured Artist song (excluding GDs), this quest isn't for you. Try scrolling down.
+
+### [Back to basics](LINK) ![bronze icon](/wiki/shared/news/2023-08-29-changes-to-the-mappers-guild/bronze-small.png)
+
+If you're too good at the game, is it hard to recognize what players find fun? It's time to find out.
+
+This quest is limited to anyone with at least **10,000 pp** in a game mode. Your goal is to rank a map with...
+
+- ...at minimum, a spread of Easy/Normal/Hard/Insane difficulties
+- ...no difficulties above 5.0 stars
+
+### [osu! original expansion](LINK) ![bronze icon](/wiki/shared/news/2023-08-29-changes-to-the-mappers-guild/bronze-small.png)
+
+Most of the [osu! originals](/wiki/osu!_originals) in osu!'s music library come from the World Cups. Many of those songs don't reach players beyond their initial mode, so let's change that.
+
+Your goal is to rank a map of any **World Cup osu! original** in a game mode that...
+
+- **...it was NOT created for.** For example, [Sanctuary](https://osu.ppy.sh/beatmapsets/1889094) was created for the *osu!* World Cup, so it's valid for *osu!taiko*/*osu!catch*/*osu!mania*.
+- **...does not have a Ranked map for the song as of June 3, 2024**. For example, [OOParts](https://osu.ppy.sh/beatmapsets/1312143) was created for the *osu!* World Cup, but also has [Ranked *osu!taiko* and *osu!mania* maps](https://osu.ppy.sh/beatmapsets?q=ooparts), so it'd only be valid for *osu!catch*.
```

Comment:
we usually don't italicise osu! because b r a n d i n g

```suggestion
- **...it was NOT created for.** For example, [Sanctuary](https://osu.ppy.sh/beatmapsets/1889094) was created for the osu! World Cup, so it's valid for osu!taiko/osu!catch/osu!mania.
- **...does not have a Ranked map for the song as of June 3, 2024**. For example, [OOParts](https://osu.ppy.sh/beatmapsets/1312143) was created for the osu! World Cup, but also has [Ranked osu!taiko and osu!mania maps](https://osu.ppy.sh/beatmapsets?q=ooparts), so it'd only be valid for osu!catch.
```